golem-cluster-dev.yaml

```diff
@@ -30,7 +30,7 @@ provider:
     node_config:
       # if not provided, image_tag will be autodetected based on currently used python and ray versions
       # check available versions at https://registry.golem.network/explore/golem/ray-on-golem
-      image_tag: "lucjandudek/ray-on-golem:py3.10.12-ray2.3.1-coremain"
+      image_tag: "lucjandudek/ray-on-golem:managers"
```

Comment:
Better tag name would be welcome

Reply:
As dev cluster copies ray-on-golem and golem-core-python over the activities, there is no need for special image.
I'm defaulting this value to non-dev value, which will be latest release.